rts/pom.xml

```diff
@@ -21,7 +21,6 @@
   <properties>
     <version.resteasy-client>${version.org.jboss.resteasy}</version.resteasy-client>
     <version.narayana>${project.version}</version.narayana>
-    <sortpom.skip>true</sortpom.skip>
```

Comment:
@marcosgopen rts/pom.xml is not a new module, I found this module `org.jboss.narayana.rts:rest-tx` in https://github.com/jbosstm/narayana/blob/5.5.34.Final/rts/pom.xml. I think you should revert the removal of `<sortpom.skip>false</sortpom.skip>`  for this module (and remove the commit with the sorted pom for this module). Is that OK for you, please?

Reply:
Sure, thanks for your review.